src/o365/spo/commands/tenant/tenant-settings-list.ts

```diff
@@ -0,0 +1,114 @@
+import auth from '../../SpoAuth';
+import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
+import * as request from 'request-promise-native';
+import config from '../../../../config';
+import commands from '../../commands';
+import Utils from '../../../../Utils';
+import {
+  CommandError
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+class SpoTenantSettingsListCommand extends SpoCommand {
```

Comment:
The command works only if you're connected to the tenant admin site. If you're connected to a regular site, you get the `Current site is not a tenant administration site.` error. We should make it explicit in the command by adding:

```ts
  protected requiresTenantAdmin(): boolean {
    return true;
  }
```

and also mention it in the command help and in the docs.

Reply:
Hi @waldekmastykarz,  I did the change. Wondering, should we add the same check for tenant-appcatalogurl-get?

Reply:
I've ran some tests and it seems like you can call `/_api/SP_TenantSettings_Current` on any URL so the change you suggested isn't necessary. Still, good of you to double check it 👍

Reply:
Good to know! Thanks!